common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -205,15 +205,6 @@ trait FeatureSwitches {
     exposeClientSide = false
   )
 
-  val QuizScoresService = Switch(
```

Comment:
I removed as many memcached references as possible, and i'll downsize the cluster